src/App.js

```diff
@@ -4,19 +4,31 @@ import data from './data/dummy.json';
 
 const App = () => {
   return (
-    <div>
-      <ol>
-        {data.map((question) => {
-          return <div>
-            <li key={question.id}>{question.text}</li>
-            <ul>{question.answers.map((answer) => {
-              return <li key={answer.id}>{answer.text}</li>
+    <form>
+      {data.map(question => {
+        return (
+          <fieldset>
+            <legend key={question.id}>{question.text}</legend>
+
+            {question.answers.map(answer => {
+              return (
+                <div>
+                  <input
+                    type="radio"
+                    id={`${question.id}.${answer.id}`}
+                    name={question.id}
+                    value={answer.id}
+                  />
+                  <label for={`${question.id}.${answer.id}`}>
```

Comment:
In HTML, `label`s use `for` but in React they use `htmlFor`

Reply:
Sorry, my bad. I didn't open console to check for errors